src/Sulu/Bundle/ContactBundle/Controller/ContactController.php

```diff
@@ -679,6 +688,30 @@ public function getSecurityContext()
         return 'sulu.contact.people';
     }
 
+    /**
+     * Takes an array of contacts and resets the avatar containing the media id with
+     * the actual urls to the avatars thumbnail.
+     *
+     * @param array $contacts
+     * @param String $locale
+     *
+     * @return array
+     */
+    private function addAvatars($contacts, $locale)
+    {
+        $ids = array_filter(array_column($contacts, 'avatar'));
+        $avatars = $this->get('sulu_media.media_manager')->getFormatUrls($ids, $locale);
+        $i = 0;
+        foreach ($contacts as $key => $contact) {
+            if (array_key_exists('avatar', $contact) && $contact['avatar']) {
+                $contacts[$key]['avatar'] = $avatars[$i];
+                $i += 1;
```

Comment:
I guess in that case I would prefer a for loop, since it is a bit cleaner:

```php
for ($i = 0; $i < count($contacts); $i++) {
    // ...
    $contacts[$i]['avatar'] = $avatars[$i];
    // ...
}
```

This is only possible if the $contacts array has numerical indices, but I think it does.